GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -200,15 +200,13 @@ protected virtual Task PostRepositoryChangedAsync()
             protected bool IsAttached { get; private set; }
             protected virtual bool SupportsFiltering { get; } = false;
 
-            public Task AttachedAsync()
+            public void Attached()
             {
                 IsAttached = true;
-                return OnAttachedAsync();
```

Comment:
I think `OnAttached()` should be kept here.

Reply:
Yes
Removed the action in RepoObjectsTree.Nodes.Submodules.cs though